crates/octez/src/rollup.rs

```diff
@@ -103,6 +103,31 @@ impl OctezRollupClient {
         }
     }
 
+    pub async fn batcher_injection<S, I>(&self, external_messages: I) -> Result<()>
+    where
+        I: IntoIterator<Item = S>,
+        S: AsRef<[u8]>,
+    {
+        let res = self
+            .client
+            .post(format!("{}/local/batcher/injection", self.endpoint))
+            .json(
+                &external_messages
+                    .into_iter()
+                    .map(hex::encode)
+                    .collect::<Vec<String>>(),
+            )
+            .send()
+            .await?;
+
+        if res.status() == 200 {
+            // TODO: Should we ignore the response?
```

Comment:
I don't think we should ignore it - the client should know if the operation wasn't successful.

Reply:
Yes, this is captured by the `else` part of this `if` block. The comment here refers to whether we should be ignoring the response from the rollup (which IIRC is a list of operation hashes)